### PR TITLE
Ets 372 phac slack reporting

### PR DESCRIPTION
**Pull request by ashtonmeuser**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "husky": "4.2.5",
     "jest": "25.3.0",
     "lint-staged": "10.1.4",
-    "nodemon": "2.0.2",
+    "nodemon": "^2.0.4",
```
Comment (mauriciofreshw) [inline server/package.json:41]: package not locked

File: server/lambda/layer/common/nodejs/custom_modules/post-to-slack.js [hunk 2]
```diff
@@ -17,33 +17,43 @@ const generateStreamLink = () => {
...
-      "text": {
-        "type": "mrkdwn",
-        "text": "Lambda Job: *${title}*\nFunction Name: ${process.env.AWS_LAMBDA_FUNCTION_NAME}:${process.env.AWS_LAMBDA_FUNCTION_VERSION}\nElapsed Time: ${time}\n\nFull output included below."
+  const escapedMessge = typeof message === 'string' ? message : JSON.stringify(message);
+  const payload = {
+    blocks: [{
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text: `Lambda Job: *${title}*\nLambda Function Name: ${formatFunctionName()}\nElapsed Time: ${time}\n\nFull output included below.`,
```
Comment (mauriciofreshw) [inline server/lambda/layer/common/nodejs/custom_modules/post-to-slack.js:35]: The PHAC Submission slack notification is coming with a Lambda Job label despite not being one, probably a more general name here?

File: server/package.json (hunk 1 above)
Comment (ashtonmeuser) [reply server/package.json:41]: Good catch. Fixed.

File: server/lambda/layer/common/nodejs/custom_modules/post-to-slack.js (hunk 2 above)
Comment (ashtonmeuser) [reply server/lambda/layer/common/nodejs/custom_modules/post-to-slack.js:35]: Good call. Fixed

**Pull request merged**